brands/shop/hardware.json

```diff
@@ -83,6 +83,20 @@
         "shop": "hardware"
       }
     },
+    {
+      "displayName": "Robert Dyas",
+      "id": "robertdyas-6506bb",
+      "locationSet": {"include": ["gb"]},
+      "matchTags": ["shop/electronics"],
+      "oldid": "shop/hardware|Robert Dyas",
```

Comment:
Should I have changed this or left it as shop/electronics?

Reply:
It doesn't really matter, the `oldid` only exists as a claim on Wikidata, and I'll probably end up bulk updating them anyway.